### PR TITLE
[CLEANUP] Drop redundant README section on installation

### PR DESCRIPTION
**Pull request by oliverklee**
Now there is only one section explaining how to install Emogrifier
with Composer, not two.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -118,38 +118,6 @@ calling the `emogrify` method:
...
-You can also add follow lines to your `composer.json` and run the
-`composer update` command:
-
-```json
-"require": {
-  "pelago/emogrifier": "^2.2"
-}
-```
-
-See https://getcomposer.org/ for more information and documentation.
```
Comment (JakeQZ) [inline README.md]: I agree we don't need to document how to use Composer.  But is it still worth retaining the link to getcomposer.org – moving it to the end of the "Installation" section, and perhaps adding "… on Composer" to the end of the sentence?

File: README.md [hunk 2]
```diff
@@ -118,38 +118,6 @@ calling the `emogrify` method:
 * `$emogrifier->addExcludedSelector(string $selector)` - Keeps elements from
   being affected by emogrification.
 
-## Installing with Composer
```
Comment (JakeQZ) [inline README.md]: This heading also needs reming from the Table of Contents.

Comment (oliverklee) [reply README.md]: Done.

File: README.md (hunk 1 above)
Comment (oliverklee) [reply README.md]: Agreed. Done.

**Comment by oliverklee** [comment]
Updated and repushed.

**Pull request merged**